<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>163</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>多项式拟合</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>12</y>
     <width>403</width>
     <height>142</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>要生成的测试数据的数量:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditNum">
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>范围:[1-100]的整数</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>进行拟合的多项式系数:  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPower">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> 范围:[0-15]的整数 </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="isNormal">
        <property name="text">
         <string>使用正则化</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lambdaInfo">
        <property name="text">
         <string>      正则化系数:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditLambda">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>0.003</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonStart">
      <property name="text">
       <string>执行</string>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/images/cartoon4.ico</normaloff>:/images/cartoon4.ico</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonErrStatistic">
      <property name="text">
       <string>查看不同多项式拟合的错误率</string>
      </property>
      <property name="icon">
       <iconset resource="pics.qrc">
        <normaloff>:/images/cartoon3.ico</normaloff>:/images/cartoon3.ico</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
